### ISSUE TITLE
multiple fill between in same plot

### ISSUE THREAD

**Issue by nima555ir** [matplotlib/mplfinance#218]
hi

i want to draw two fill between in a same chart .
for example in ichikomu indicator, when SpanA is less than SpanB the fill between color set to green and in reverse the color set to blue.

i noticed that the fill between argument in plot method only accept one dict and condition. how it is possible two have two different fill between in same chart with different color and condition?

thanks

**Comment by DanielGoldfarb** [comment]
That's correct, `fill_between` kwarg currently accepts only one fill-between specification (number, sequence, or dict).  (Although that one fill-between _can_ be applied to any "panel", it is still limited to one specification).

Definitely we want to enhance that.  I've been debating whether to ...

- accept `fill_between` also as a **list** of  specifiers (similar to how `addplot` accepts a single specification or a list), or
- support `fill_between` kwarg in method `mpf.make_addplot()` (as the way to specify multiple fill-betweens).

Thoughts?  Pros/Cons for each approach?

**Comment by nima555ir** [comment]
i think the second scenario is better because it makes possible to have multiple fill_between specifiers for various panels.

thanks alot

**Comment by fxhuhn** [comment]
I also suppose that with the second variant it is possible to show the positive and negative cloud of the ichikomu indicator.

**Comment by DanielGoldfarb** [comment]
@fxhuhn Markus, yes, once multiple fill betweens are allowed you should be able to show positive and negative ichikomu clouds (although I imagine you may have to specify the fill betweens separately for the positive and negative.  I'll keep that in mind when implementing multiple fill betweens and try to make it easy.

**Comment by IanAtDazed** [comment]
Is there any update on when multiple fill_betweens might be available?

Thanks!

**Comment by lambolambert** [comment]
hows this one coming along?

**Comment by DanielGoldfarb** [comment]
@lambolambert 

This is done. 
See bottom of following link for usage details:
https://github.com/matplotlib/mplfinance/blob/master/examples/fill_between.ipynb

**Comment by DanielGoldfarb** [comment]
P.S. make sure you have installed the most recent version.
